### PR TITLE
Update Issue Moderation Guide

### PR DESCRIPTION
**Pull request by SaintPeter**
Massive rewrite of the Issue Mod guide.

I've tried to be exhaustive, but have no doubt missed key topics.  Would love feedback/corrections/additions.

tag: @FreeCodeCamp/issue-moderators

### CODE REVIEW

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 1]
```diff
@@ -1,16 +1,72 @@
...
+2. Must be from a properly named non-staging branch on the user's fork
+3. Title must clearly identify the affected area/change made
+4. Title should NOT have an issue number in it
+5. Body of the PR should give details about the change as well as level of testing (IE: untested, tested locally)
+6. If the PR is in response to an open issue or issues, the body should also include `closes #XXXX` for each issue number closed
+7. Change should only have 1 commit
+8. Code should pass all tests and linting
+9. Code should be of general high quality and a needed change or improvement
+
+If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines.  New Contributors may be referred to the HelpContributors Chat room.  At at the Mod's discretion the issue may be closed.
```
Comment (sludge256) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: Link the the HelpContributors room

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 2]
```diff
@@ -1,16 +1,72 @@
...
+
+If you are approved, we will add you to a GitHub Team, `Issue mods`.
 
-Also note, the number of issue moderators will always remain small due to the nature of Github permissions.
+The number of issue moderators will always remain small due to the nature of Github permissions.
 
-Happy Coding!
+#### Additional Requirements
+* Two Factor Authentication enabled on your GitHub account
+* Profile Name set to at least you first name
```
Comment (sludge256) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: You should be your.

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 3]
```diff
@@ -1,16 +1,72 @@
...
+#### Issue Moderation
+Free Code Camp is an active open source project.  We get many tens of issues a day, all of which need to be triaged and labeled.
 
-If you are approved, we will add you to a GitHub Team, `Issue mods`.
+ There are several general classes of Issues:
+1. **Code Help Requests**  
+Help Requests are not appropriate for Issues.  Do not provide code support in your response, gently point the user to the appropriate Help Chat, and close the issue.
+2. **Bug or Clarification issues**  
+Confirm or validate the bug if possible.  Seek additional clarification or details if needed.  Once the issue has been reproduced or at least deemed legitimate, label it `confirmed`.
+    * If it's a simple change to an existing challenge, flag as `help wanted` and, optionally, as `easy`.  Use other tags as appopriate.
```
Comment (bugron) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: appopriate -> appropriate

**Comment by bugron** [comment]
Also missing periods at the end of some sentences but the text is good.

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 4]
```diff
@@ -1,16 +1,72 @@
...
+    * If the issue is more significant, flag as `bug`
+    * Other labels as appropriate
+
+    If there is a question as to the proper course of action on an issue, feel free to call @issueModerators to give an opinion.  Flag as `Discussing`.
```
Comment (bugron) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: @issueModerators -> @FreeCodeCamp/issue-moderators

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 5]
```diff
@@ -1,16 +1,72 @@
...
+PRs which change the underlying function of the site or make non-trivial changes to the UI or UX of the site should be approved by @BerkeleyTrue or @QuincyLarson.  If you have any doubt, tag them in a comment and/or draw their attention to the PR via Gitter Chat.
+
+##### Take Care
 Though you will have write access to Free Code Camp's repository, **you should never write directly to the Free Code Camp repository**. All code should enter Free Code Camp's codebase in the form of a pull request.
 
-Please note that we will frequently remove issue mods whom we think are inactive. If you are removed, please do not take this personally - we can add you back to the team, just message us and let us know you're still active.
+## Becoming an Issue Mod
+In order to become an issue mod, you must first prove your helpfulness by leaving useful comments on outstanding issues, and submitting pull requests of your own to fix these issues.
+
+If you've been doing these things, and want the additional power/responsibility that comes with helping Free Code Camp as an issue mod, please contact [@berkeleytrue](https://gitter.im/berkeleytrue) in Gitter.
```
Comment (bugron) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: `@berkeleytrue` -> `@BerkeleyTrue`

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md (hunk 1 above)
Comment (sludge256) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: Link to the HelpContributors room

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md (hunk 2 above)
Comment (sludge256) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: You should be your

File: How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md [hunk 6]
```diff
@@ -1,16 +1,72 @@
...
+
+##### Quality Assurance
+Assuming that the basic requirements for the PR are met, all PRs should undergo some level of Quality Assurance testing.  The most basic QA is to simply checkout the PR on a local copy of the site and test the changed functionality.  Be sure to read through the code changes and understand what the potential side effects are. Exercise code and corner cases.  Be sure to try both negative and positive test cases.
 
+If there is any doubt about the functionality, ask for the @issueModerators to also take look.
+
+For larger PRs, tag in @BerkeleyTrue.  In some cases it may make sense to test on the beta site.
+
+##### Special Requirements
+PRs which change the underlying function of the site or make non-trivial changes to the UI or UX of the site should be approved by @BerkeleyTrue or @QuincyLarson.  If you have any doubt, tag them in a comment and/or draw their attention to the PR via Gitter Chat.
```
Comment (bugron) [inline How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md]: Maybe `@BerkeleyTrue` -> Berkeley's Gitter acc and `@QuincyLarson` -> Quincy's Gitter acc?

**Comment by bugron** [comment]
@SaintPeter it would be nice to have a link in this article to our team: https://github.com/orgs/FreeCodeCamp/teams/issue-moderators.

**Comment by SaintPeter** [comment]
I didn't even know we HAD a team page.  I'll make the edits tomorrow via the GitHub interface.  I was doing it offline and my client side render was obviously not identical to GitHubs.

**Comment by bugron** [comment]
Also a short info about labels https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Select-Issues-for-Contributing-Using-Labels and we need to update that article too.

**Comment by SaintPeter** [comment]
Requested edits and additions made.

**Comment by bugron** [comment]
Awesome, merged!
